### PR TITLE
Web schedules fixes bunch 3

### PR DESCRIPTION
**Pull request by maskin25**
**What this PR does**:

Polish calendar web UI up, add Rotation create/update visual helper

**Which issue(s) this PR fixes**:

https://github.com/grafana/oncall/issues/532

**Checklist**
- [ ] Tests updated
- [ ] Documentation added
- [ ] `CHANGELOG.md` updated

### CODE REVIEW

File: grafana-plugin/src/containers/RotationForm/RotationForm.tsx [hunk 1]
```diff
@@ -154,7 +154,7 @@ const RotationForm: FC<RotationFormProps> = observer((props) => {
       rolling_users: userGroups,
       interval: repeatEveryValue,
       frequency: repeatEveryPeriod,
-      by_day: repeatEveryPeriod === 1 || repeatEveryPeriod === 0 ? selectedDays : null,
+      by_day: (repeatEveryPeriod === 0 && repeatEveryValue === 1) || repeatEveryPeriod === 1 ? selectedDays : null,
```
Comment (matiasb) [inline grafana-plugin/src/containers/RotationForm/RotationForm.tsx:157]: FYI, if I understand this correctly, it limits the repeat input value to 1 but the backend allows to choose a different value as long as the number is less than or equal to the length of selected days (same below).

**Review by matiasb — APPROVED** [review]
LGTM, added one minor comment

File: grafana-plugin/src/containers/RotationForm/RotationForm.tsx (hunk 1 above)
Comment (maskin25) [reply grafana-plugin/src/containers/RotationForm/RotationForm.tsx:157]: > length of selected days

but if you select  days not in a row (e.g. Mo We and Su) and how it should depend on shift start and shift end? It's good topic to discuss, there are still some questions

Comment (maskin25) [reply grafana-plugin/src/containers/RotationForm/RotationForm.tsx]: I've removed `repeatEveryValue === 1` checking

**Pull request merged**